runtime/src/bank_forks.rs

```diff
@@ -17,7 +17,8 @@ use {
     },
 };
 
-struct SetRootTimings {
+#[derive(Debug, Default, Copy, Clone)]
+struct SetRootMetrics {
```

Comment:
Maybe `SetRootMetrics` can contain the original `SetRootTimings` for organization? These structs tend to become a dumping ground of random fields otherwise.

Reply:
Sure, I can do that. I actually originally was going down that path, then I saw that `SetRootTimings` already contained `tx_count` and `dropped_banks_len`. Would you like me to move those two fields _out_ of Timings and into Metrics?

Reply:
Done!